lib/src/rules/avoid_setters_without_getters.dart

```diff
@@ -73,8 +73,8 @@ class _Visitor extends SimpleAstVisitor<void> {
 
   @override
   void visitClassDeclaration(ClassDeclaration node) {
-    final methods = node.members.where(isMethod);
-    for (MethodDeclaration method in methods) {
+    for (var member in node.members.where(isMethod)) {
```

Comment:
@munificent It might be nice if pattern matching/destructuring provided a nicer way to handle this idiom.